docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Let's remove this page since we added the link to the new guide to `docs/pages/setup/helm-reference`.